packages/ripple-binary-codec/HISTORY.md

```diff
@@ -1,6 +1,7 @@
 # ripple-binary-codec Release History
 
 ## Unreleased
+- Small typing refactor in Shamap to match newer version of TypeScript.
```

Comment:
Does this affect the exported methods, or it just internal? If it's just internal, then it doesn't need to be in the changelog - it's effectively just a refactor.

Reply:
Just internal, shouldn't affect anything